server/src/main/java/org/elasticsearch/index/codec/PerFieldFormatSupplier.java

```diff
@@ -77,7 +77,7 @@ boolean useBloomFilter(String field) {
             // but based on dimension fields and timestamp field, so during indexing
```

Comment:
I think the `excludeFields()` method in this field also needs to be updated? Otherwise `_logs_id` would use the default lucene doc values codec?

Reply:
Good catch, updated.